### DISCUSSION TITLE
Querings between dates / timestamps

### DISCUSSION THREAD

**Discussion by jordancrubin** [deta/cloud-docs#675] [Help]
I only see mention of expiration dates, so using that format I created a datetime value.  

"items":[{"date":"2022-12-21T16:34:45","event":"OPEN","zone":"1111111","trigger":1,"meter":77}]

How would one query a date, as there is no datetime or timestamp data value

ALL ON dec 21 2022
ALL in dec 2022
all in 2022
between nov 15 and dev 15 2022

There seems to be no documentation on this

**Comment by lemonyte** [comment]
Querying by date is not possible, Deta Base works with basic types such as strings, integers, and lists. You can try to create your own logic to query by UNIX timestamp. For example, "all in December 2022" is `{"date?gte": 1669870800", "date?lte": 1672549199}`.
